### DISCUSSION TITLE
我想知道怎么按地区测速ipv6

### DISCUSSION THREAD

**Discussion by flylj999** [XIU2/CloudflareSpeedTest#557] [提问]
我用代码 CloudflareST.exe -f ipv6.txt -cfcolo TWN ，可出来的好像还是香港的，也就是所有ipv6里最快的，并没有选择台湾地区

**Comment by XIU2** [comment]
#542 

另外，你忘记加上 `-httping` 参数了，因为 `-cfcolo` 参数只有在 HTTPing 测速模式下才有效

这点是我疏忽了，忘记在示例命令里加上 `-httping` 了（我只写了 ...只有在 HTTPing 延迟测速模式下才可用...）

**Reply by flylj999** [reply]
啊 看到了，是横屏太长 后面盖住了 哈哈哈 等会去试试，哦 下面有 没注意看 哈哈 不懂这些专业术语。另外，为什么ipv6有4万多，ipv4才5000多 测试时间差好多啊

**Reply by flylj999** [reply]
测了 台湾 和新加坡 可用都是0 这是啥原因啊

**Reply by XIU2** [reply]
如果延迟测速后结果为 0，则说明没有找到任何一个（未超时可用的）指定地区的 IP。

**Reply by flylj999** [reply]
这么惨 台湾 新加坡 好像挺常用 常见的啊

**Reply by XIU2** [reply]
这是又 Cloudflare CDN 决定的，你可以了解一下什么是 Anycast IP。

不同地区、不同运营商、不同时间段访问同一个 IP 分配到的服务器节点地区和路线也都是不一样的（比如同一个 IP，美国人访问就是分配到就近的美国节点服务器，日本人访问则就又变成了就近的日本节点服务器了，当然不同的 IP 段路由变化/分配逻辑也是不同的）。

如果你在台湾，那么你随便测速都是一大片的台湾节点，但是你在中国大陆，那么就很难遇到，只有通过寻找一些特殊的较为固定的 IP 段才行。
国内最常见的还是分配到美国、香港（主要是移动路线），分配到其他地方的都很罕见，需要靠运气。

而且 Anycast IP 的路由是经常变的，同一个 IP 今天可能是美国，明天你再访问可能就又分配到欧洲节点了（当然这只是个例子，一般没有那么频繁，这也和很多因素有关，比如线路拥塞程度，成本变动等）。

**Comment by flylj999** [comment]
是的 我测的都是香港的 IPv6比ipv4还快点。但是Gemini这类ai不让用 想换个ipv6试试呢在 2024年12月6日，21:24，XIU2 ***@***.***> 写道：﻿
这是又 Cloudflare CDN 决定的，你可以了解一下什么是 Anycast IP。
不同地区、不同运营商、不同时间段访问同一个 IP 分配到的服务器节点地区和路线也都是不一样的（比如同一个 IP，美国人访问就是分配到就近的美国节点服务器，日本人访问则就又变成了就近的日本节点服务器了，当然不同的 IP 段路由变化/分配逻辑也是不同的）。
如果你在台湾，那么你随便测速都是一大片的台湾节点，但是你在中国大陆，那么就很难遇到，只有通过寻找一些特殊的较为固定的 IP 段才行。
国内最常见的还是分配到美国、香港（主要是移动路线），分配到其他地方的都很罕见，需要靠运气。
